extensions/wikia/AssetsManager/config.php

```diff
@@ -2211,14 +2211,20 @@
 	],
 ];
 
-$config['on_site_notifications_js'] = [
+$config['design_system_on_site_notifications_js'] = [
 	'type' => AssetsManager::TYPE_JS,
 	'skin' => [ 'oasis' ],
 	'assets' => [
 		'//extensions/wikia/DesignSystem/scripts/DesignSystemGlobalNavigationOnSiteNotifications.js'
 	],
 ];
 
+$config['design_system_on_site_notifications_mustache_js'] = [
+	'type' => AssetsManager::TYPE_JS,
+	'skin' => [ 'oasis' ],
+	'assets' => [],
```

Comment:
So this is not a placeholder, rather than that, the JS gets loaded by the mustache loader. I would have included this in the `design_system_on_site_notifications` but it causes it to be reloaded and therefore re-initialised again and again (because this in turn loads mustache again).